src/lando/main/models/profile.py

```diff
@@ -46,9 +46,11 @@ def filter_claims(claims: dict) -> dict:
     # NOTE: currently these are SCM group, however in the future other
     # groups will need to be added here, for example "treestatus users",
     # and other lando permissions.
-    claims[CLAIM_GROUPS_KEY] = [
-        group for group in claims[CLAIM_GROUPS_KEY] if "scm" in group.lower()
-    ]
+    # This filter is not applicable to local development.
+    if CLAIM_GROUPS_KEY in claims:
```

Comment:
Maybe we should check for `settings.ENVIRONMENT == Environment.local` here instead, since the comment suggests that's the actual condition we want to check.